Lodgify.Payments.Stripe.Server.IntegrationTests/Factories/CustomWebApplicationFactory.cs

```diff
@@ -73,33 +83,16 @@ public async Task InitializeAsync()
         WiremockClient = _wireMockContainer.CreateWireMockAdminClient();
 
         await MigrateDatabaseAsync<PaymentDbContext>();
-
         await SeedDatabaseAsync();
     }
 
-    public async Task DisposeAsync()
+    public new async Task DisposeAsync()
     {
         await _postgresContainer.DisposeAsync();
         await _wireMockContainer.DisposeAsync();
         _scope.Dispose();
     }
```

Comment:
Not fully sure but isn't `new` makes that WebApplicationFactory.DisposeAsync may be not executed?

If that's a case how about explicit implementation of IAsyncLifetime interface here?
```suggestion
    async Task IAsyncLifetime.DisposeAsync()
    {
        await _postgresContainer.DisposeAsync();
        await _wireMockContainer.DisposeAsync();
        _scope.Dispose();
    }
```